src/config.rs

```diff
@@ -704,6 +737,39 @@ fn config_from_env() -> Result<EnvConfig> {
         None
     };
 
+    // ======= Buildless =======
+    let buildless = if env::var("SCCACHE_BUILDLESS").is_ok() || env::var("BUILDLESS_APIKEY").is_ok()
+    {
+        // read kill switch
+        let enable_var = env::var("SCCACHE_BUILDLESS").ok().unwrap_or_default();
```

Comment:
buildless takes last precedence over all other adapters (except fallback to local disk), to prevent conflicts with providers which expect to fail with incomplete config.